frontend-mobile/lib/view/settings_screen.dart

```diff
@@ -1,16 +1,47 @@
+import 'package:coffee_caller/storage/settings_storage.dart';
 import 'package:flutter/material.dart';
 
-class SettingsScreen extends StatelessWidget {
+class SettingsScreen extends StatefulWidget {
   const SettingsScreen({super.key});
 
+  @override
+  State<StatefulWidget> createState() => _SettingsScreenState();
+}
+
+class _SettingsScreenState extends State<SettingsScreen> {
+  final TextEditingController _usernameController = TextEditingController();
+
+  @override
+  void initState() {
+    super.initState();
+    getUsername().then((value) => _usernameController.text = value);
```

Comment:
Apparently flutter does not support async in `initState`, should I switch to a FutureBuilder?

Reply:
I think this is fine. When loading more than one value, it's time to introduce a bloc, but for now we're good.